ouroboros-network/src/Ouroboros/Network/Testing/ConcreteBlock.hs

```diff
@@ -345,3 +350,21 @@ instance Serialise BlockBody where
   encode (BlockBody b) = encodeString (Text.pack b)
 
   decode = BlockBody . Text.unpack <$> decodeString
+
+{-------------------------------------------------------------------------------
+  Simple static time conversions, since no HardFork
+-------------------------------------------------------------------------------}
+
+-- no HFC, so no side-effects necessary
+convertSlotToTimeNoHardFork :: SlotNo -> UTCTime
+convertSlotToTimeNoHardFork sl =
```

Comment:
We might want to be more extreme in the comment and the name of this to avoid people copying it from demos/exmples.

Reply:
Thanks, pushed it 👍